data/lua/wml/modify_unit.lua

```diff
@@ -135,6 +136,11 @@ local function simple_modify_unit(cfg)
 				end
 				u:add_modification(tagname, tagcontent);
 			end
+			if tagname == "status" then
```

Comment:
Why is this needed? Isn't it handled by the general case?

Reply:
well this is,  like the whole simple_modify_unit function kinda an optimisation over the 'general' case which does store+unstore_unit.